db/chief_cds_guidance.json

```diff
@@ -1,4 +1,5 @@
 {
+    "last_update": "2022-06-16",
```

Comment:
This is to keep track of the last update, it is not used by the code.

Reply:
Maybe just remove the last_update date. We know the file changed because github tells us

Reply:
👍